frontend/src/components/MultiSelect.js

```diff
@@ -150,7 +150,7 @@ function MultiSelect({
             isDisabled={disabled}
             tabSelectsValue={false}
             isClearable={multiSelectOptions.isClearable}
-            closeMenuOnSelect={multiSelectOptions.closeMenuOnSelect}
+            closeMenuOnSelect={multiSelectOptions.closeMenuOnSelect || false}
```

Comment:
@jasalisbury do we need an OR for 'false' or will closeMenuOnSelect default to 'false' if its null or undefined? Not sure if this was just a safety net.